integrations/adding-an-integration/connecting-to-slack.md

```diff
@@ -0,0 +1,46 @@
+# Connecting to Slack
+
+You can connect aqueduct to your Slack App in order to receive automated notifications on your workflow execution status. You can simply click the slack icon on your [integrations page](../../integrations.md). Aqueduct will ask for [#Bot token](connecting-to-slack#setting-up-slack-app), [#channels](connecting-to-slack#configuring-channels), and [#workflow](connecting-to-slack#configuring-global-workflow-settings) information to connect to an email integration.
+
+## Setting up Slack App
+To connect Aqueduct to slack to recieve slack notifications, you would first need to set up your own Slack App. You can follow these steps to create one:
+* Go to [Slack Apps management page](https://api.slack.com/apps).
+* Hit **Create New App** button.
+<figure><img src="../../.gitbook/assets/connecting_slack_create_app.png" alt=""><figcaption></figcaption></figure>
+
+* Select **Create from manifest** on the dialog.
+* Select a workspace you would like to enable notification.
+* Choose **yaml** and copy over the following content to the manifest. You can change **name** and **display_name** fields. This step sets up minimum required permissions and configurations for this app. You can add additional settings once the creation is done.
```

Comment:
Nit: Is there any scenario in which they'd want to add additional settings? If not, let's remove that.

Reply:
I think the user could use the same app for other notification purposes, but I agree mentioning 'additional settings' could be confusing and will remove